net-news/liferea/liferea-1.12.5.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{4,5,6,7} )
+
+inherit gnome2 python-single-r1
+
+MY_P=${P/_/-}a
```

Comment:
Any reason to hack it around instead of adding `a` to PV?

Reply:
It was released by upstream as ["v1.12.5"](https://github.com/lwindolf/liferea/releases/tag/v1.12.5), so I decided to not call it "1.12.5a".

Reply:
1. …which introduces unnecessary complexity to the ebuild which will have to be reverted on next bump.
2. …which confuses [repology](https://repology.org/metapackage/liferea/versions) which will claim that Gentoo version is outdated.

Reply:
Looking to repology - AOSC and "SlackBuilds" maintainers are only persons who confused this web resource.  
Alt, Arch, Debian, Devuan, Fedora, Kali, Manjaro.... and others set package version as released by maintainer - "1.12.5".  
If you will not insist on naming change, I will handle revert in the next Liferea release.

Reply:
I looked to Liferea's "About" page and version here is "1.12.5a". Looks like there is no any author reasons for "1.12.5" tag.  
I will change ebuild name.